django-entrypoint.sh

```diff
@@ -14,11 +14,13 @@ if [ "$1" = 'celery' ]; then
 fi
 
 if [ "$1" = 'gunicorn' ]; then
-  # Do a chown of the /app/media directory (if it exists) at runtime in case the
-  # directory was mounted as a root-owned volume.
-  if [ -d /app/media ] && [ "$(stat -c %U /app/media)" != 'django' ]; then
-    chown -R django:django /app/media
-  fi
+  # Do a chown of the /app/media & /app/mediafiles directories (if they exist)
+  # at runtime in case the directory was mounted as a root-owned volume.
+  for media in /app/media /app/mediafiles; do
+    if [ -d $media ] && [ "$(stat -c %u $media)" != "$(id -u django)" ]; then
```

Comment:
Can you make it like above (`%U`, `!= 'django'`, etc)? I copy pasted from the code you linked which was slightly old.